docs/design/shifu-framework/telemetry-service-refactor.md

```diff
@@ -0,0 +1,95 @@
+# TelemetryService Refactor
+
+## What is TelemetrySerice
+
+***deviceShifu*** Telemetry is part of the ***deviceShifu*** [Compute](docs/design/design-deviceShifu.md#compute) plugin. It periodically pulls device instructions defined in the telemetries section in ***deviceShifu*** ConfigMap.
+
+The result for each telemetry should update `EdgeDevice`'s status accordingly.
+
+## Why need to refactor TelemetryService?
+
+- Currently, telemetryServices' connection settings are read from deviceShifu, which will send all connection settings to telemetryService, so telemetryService strongly depends on deviceShifu.
+- telemetryService's connection settings may include some private information, such as secrets read from deviceShifu, so that deviceShifu will have more access to read this information from the kubernetes API, which will make users feel scared.
+- The installation yaml file of Shifu needs to be separate from the yaml file of telemetryService.
```

Comment:
Don't think we need to separate these? They are just part of the Shifu CRD

Reply:
not CRD, but like cluster role clusterrolebinding